plugins/woocommerce/tests/e2e-pw/tests/merchant/create-variable-product.spec.js

```diff
@@ -251,7 +241,28 @@ test.describe( 'Add New Variable Product Page', () => {
 	} ) => {
 		await page.goto( productPageURL );
 		await page.fill( '#title', manualVariableProduct );
-		await page.selectOption( '#product-type', 'variable', { force: true } );
+		await page.selectOption( '#product-type', 'variable' );
+
+		await page
+			.locator( '.attribute_tab' )
+			.getByRole( 'link', { name: 'Attributes' } )
+			.scrollIntoViewIfNeeded();
+
+		// the tour only seems to display when not running headless, so just make sure
+		if ( await page.locator( '.woocommerce-tour-kit-step__heading' ).isVisible() ) {
```

Comment:
Updated the selector, but otherwise it's a copy/paste of the code from the first "test".  Ensure that the tour is dismissed before continuting.